libs/platform/src/lib/components/approval-flow/approval-flow-add-node/approval-flow-add-node.component.ts

```diff
@@ -1,18 +1,12 @@
 import { ChangeDetectionStrategy, ChangeDetectorRef, Component, OnDestroy, OnInit, TemplateRef } from '@angular/core';
-import { DialogRef, FdDate } from '@fundamental-ngx/core';
 import { forkJoin, Observable, Subscription } from 'rxjs';
 import { take } from 'rxjs/operators';
 
-import {
-    ApprovalDataSource,
-    ApprovalNode,
-    ApprovalTeam,
-    ApprovalUser
-} from '../public_api';
-import {
-    ApprovalFlowAddNodeViewService,
-    VIEW_MODES
-} from '../services/approval-flow-add-node-view.service';
+import { FdDate } from '@fundamental-ngx/core/datetime';
+import { DialogRef } from '@fundamental-ngx/core/dialog';
+
+import { ApprovalDataSource, ApprovalNode, ApprovalTeam, ApprovalUser } from '../public_api';
```

Comment:
I think we should be referencing these files directly [than through `public_api`](https://github.com/SAP/fundamental-ngx/pull/4889#issue-588212919)